src/MarginTrading.Backend/Controllers/DayOffExclusionsController.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using JetBrains.Annotations;
+using MarginTrading.Backend.Contracts.DayOffSettings;
+using MarginTrading.Backend.Core.DayOffSettings;
+using MarginTrading.Backend.Services.AssetPairs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MarginTrading.Backend.Controllers
+{
+    [Route("api/[controller]")]
+    public class DayOffExclusionsController : Controller
```

Comment:
I think controller should implement corresponding client to avoid mistakes. Even if we need to call Task.FromResult everywhere.

Reply:
I'll try my best to do it then

Reply:
Done